docs/basic-features/data-fetching.md

```diff
@@ -727,6 +727,12 @@ The `context` parameter is an object containing the following keys:
 >
 > Fetching from an external API is fine!
 
+### Built-in `context.req` parsing
+
+To improve the developer experience, the `context.req` request object received by `getServerSideProps` is pre-parsed to include some additional helpful properties. Those properties are:
```

Comment:
```suggestion
The `req` in the context passed to `getServerSideProps` provides built in middleware that parses the incoming request (req). That middleware is:
```

Tweaked the wording a bit to try to match closer to https://nextjs.org/docs/api-routes/api-middlewares